core/dbt/cli/params.py

```diff
@@ -373,10 +374,20 @@
     default="{}",
 )
 
+
+def version_callback(ctx, param, value):
```

Comment:
I'm not sure if this is the best place for this function, but I wanted to use click's callbacks for this since we're already so invested in click.